src/test-utils/composeMockAppTree/index.jsx

```diff
@@ -27,14 +29,23 @@ const Index = (statePreset = 'preOnboard', options = {}) => {
 
   const Tree = ({ children }) => {
     return (
-      <WalletProviderWrapper
-        options={options}
-        statePreset={statePreset}
-        getState={cacheWalletProviderState}
-        initialState={initialState}
+      <Environment
+        coinType={CoinType.TEST}
+        networkName={Network.CALIBRATION}
+        nodeStatusApiKey='m787669344-2a9b90eb03dbff3e503c93c7'
+        graphUrl='graph-calibration.glif.link/query'
+        lotusApiUrl='https://api.calibration.node.glif.io/'
+        explorerUrl='https://explorer-calibration.glif.link'
```

Comment:
Yeah, the `Tree` just renders a lot of stuff in the test environment that isn't always necessary for certain tests. For instance, the CIDLink tests shouldn't need the wallet-provider, balance poller...etc - feels like that could make the tests a lot slower and more clunky

I do agree though, it would be nice to have a standard test environment to standardize all tests across all repos. The current one is pretty messy and could be improved and streamlined. Feels like i could advocate for that being a separate follow on PR in the future though. Since for now, i don't see these changes (albeit a bit repetitive) as too or any harder to maintain, since we're not constantly adding new tests to old components? 

Happy to be convinced otherwise though, if you feel strongly we should revise the test env in this PR i can work through it